src/core/File.ts

```diff
@@ -222,7 +220,7 @@ export class File {
                 } else {
                     itemTest = item.test;
                 }
-                if (itemTest && utils.isFunction(itemTest.test) && itemTest.test(path.relative(appRoot.path, this.absPath))) {
```

Comment:
I think it has to be appRoot, otherwise it might happen that node_modules are not found. Please revert.